### DISCUSSION TITLE
Installing on Ubuntu

### DISCUSSION THREAD

**Discussion by PRA7H1K** [kiki7000/discord.py-components#14] [Q&A]
I installed discord_buttons on Windows and it worked fine but when I try installing it on Ubuntu, it says "Could not find a version that satisfies the requirement discord_buttons (from versions: ) No matching distribution found for discord_buttons". Does anyone know how to install discord_buttons on Ubuntu?

**Comment by kiki7000** [comment]
Please install `discord-components` not `discord-buttons`
`discord-buttons` is outdated

**Reply by PRA7H1K** [reply]
oh ok. I just tried installing discord-components and the same error comes up.

**Reply by PRA7H1K** [reply]
![discord-components not installing](https://user-images.githubusercontent.com/64102705/119831553-652cb780-becb-11eb-8d9e-60982e26f48b.JPG)

**Reply by PythonSerious** [reply]
Hey!, You need to upgrade python3.6 to python3.9

**Reply by PythonSerious** [reply]
Just google it and it should sort you out!

**Comment by jehanv** [comment]
I believe you also need to `pip install discord`

**Reply by kiki7000** [reply]
discord will be automaticly installed
